Dockerfile

```diff
@@ -19,7 +17,7 @@ ENV LANG C.UTF-8
 RUN apt-get update \
   && apt-get install --yes --no-install-recommends \
     apt-utils \
-    createrepo \
+    createrepo-c \
```

Comment:
Doesn’t this create a separate command than the old package?
Do the tests cover creating a repo?

Reply:
Oh good point. I added a symlink as a compatibility shim to help us transition. Once this image is adopted, callers can be adjusted to call `createrepo_c`, after which the compatibility shim can be removed.